File: algorithms/count-apples-oranges/countApplesOranges.php
<?php
function countApplesAndOranges($s, $t, $a, $b, $apples, $oranges) {
  $landApple = 0;
  $landOrange = 0;

  foreach ($apples as $apple) {
    if (($a + $apple >= $s) && ($a + $apple <= t)) {
      $landApple++;
    }
  };

  foreach ($oranges as $orange) {
    if (($b + $orange >= $s) && ($b + $orange <= $t)) {
      $landOrange++;
    }
  };

  echo "$landApple\n$landOrange";
}
?>

File: algorithms/count-apples-oranges/count_apples_oranges.py
def countApplesAndOranges(s, t, a, b, apples, oranges):
  land_apple = int()
  land_orange = int()

  for apple in apples:
    if (s <= a + apple <= t):
      land_apple += 1

  for orange in oranges:
    if (s <= b + orange <= t):
      land_orange += 1
    
  print(f"{land_apple}\n{land_orange}")

